autopush/tests/test_integration.py

```diff
@@ -35,6 +35,7 @@
 
 
 def setUp():
+    logging.getLogger('boto').setLevel(logging.CRITICAL)
```

Comment:
It was generating a large amount of noise, yes. Under individual nosetests run it was in debug mode, so this is to drop it back to only show critical errors.